INSTALL.md

```diff
@@ -14,7 +14,7 @@ Clone the following two repositories:
 * https://github.com/HERA-Team/hera_cm_db_updates
 
 Then install by:
-1. within the hera_mc directory type `python setup.py install`
+1. within the hera_mc directory type `python setup.py install` or `pip install .`
```

Comment:
Do we think `pip install .` is better? If so, let's just say to use that one.

Reply:
It seems to pollute the hard drives less.  But either are fine and maybe someone doesn't have pip installed?